docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,162 @@
+
+## Install React Native Firebase
+Run the following commands to remove `expo-firebase-analytics` and install the React Native Firebase packages:
+
+<Terminal cmd={[
+'# Remove Expo Firebase packages',
+'$ yarn remove expo-firebase-analytics',
+'',
+'# Install RN Firebase, etc.',
+'$ yarn add @react-native-firebase/app @react-native-firebase/analytics'
+]} />
+
+If you are using [Expo Prebuild](/guides/adopting-prebuild) (also known as Managed Workflow) and building your app for iOS, install the `expo-build-properties` plugin and apply the following configuration to your [Expo config](/workflow/configuration/) file.
+
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
+
+### Add Config plugin to Expo config file
+
+In the Expo config file, add add `expo-community-flipper` to the `plugins` array with the following configuration:
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+  }
+}
+
+If you are using the Bare Workflow, see [React Native Firebase's installation instructions](https://rnfirebase.io/#bare-workflow).
+
+## Update analytics requests
+
+Firebase Analytics requests in React Native Firebase are almost the same- mostly, they are imported a little differently. Update your code as shown here:
+
+```diff
+// update imports
+- import * as Analytics from "expo-firebase-analytics";
++ import analytics from "@react-native-firebase/analytics";
+
+// update each request to analytics
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [Development Build](/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase.
+
+[Follow these instructions for creating a development build](https://docs.expo.dev/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
+
+```json eas.json
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    }
+  }
+}
+If you wish to run a [Development Build on an iOS simulator](/build-reference/simulators/) instead of a device, your profile may also include:
+
+```json eas.json
+{
+  "development": {
+      //..
+      "ios": {
+        "simulator": true
+      }
+    },
+}
```

Comment:
```suggestion
```json eas.json
{
  "development": {
      //..
      "ios": {
        "simulator": true
      }
    },
}
```
```

I think my suggestions yesterday messed the code snippet syntax.